functions/src/mappers/github/status.mapper.ts

```diff
@@ -22,8 +22,8 @@ export class GitHubPullRequestStatusMapper {
       id: input.id,
       state: input.state,
       context: input.context,
-      createdAt: firestore.Timestamp.fromDate(new Date(input.created_at)),
-      updatedAt: firestore.Timestamp.fromDate(new Date(input.updated_at)),
+      createdAt: input.created_at,
+      updatedAt: input.updated_at,
```

Comment:
Any reason why these have gone back to `string`s? We updated them to be compatible with firestore

Reply:
I gave the answer in above comment. Please check